packages/core/src/components/card/card-vars.scss

```
@@ -1,48 +1,80 @@
-:root,
-.tds-mode-light {
-  --tds-card-background: var(--tds-white);
-  --tds-card-box-primary: 0 3px 3px rgb(0 0 0 / 15%), 0 -1px 1px rgb(0 0 0 / 1%);
-  --tds-card-box-secondary: 0 -1px 1px rgb(0 0 0 / 5%), 0 3px 3px rgb(0 0 0 / 10%);
-  --tds-card-box: var(--tds-card-box-primary);
-  --tds-card-box-hover: 0 -1px 1px rgb(0 0 0 / 5%), 0 3px 3px rgb(0 0 0 / 20%);
-  --tds-card-box-pressed: 0 -1px 1px rgb(0 0 0 / 5%), 0 1px 3px rgb(0 0 0 / 20%);
-  --tds-card-headline: var(--tds-grey-950);
-  --tds-card-sub-headline: var(--tds-grey-500);
-  --tds-card-body-color: var(--tds-grey-500);
-  --tds-card-divider: var(--tds-grey-300);
-  --tds-card-icon-color: var(--tds-grey-950);
+tds-card {
+  --card-background: var(--tds-white);
+  --card-background-primary: var(--component-card-primary);
+  --card-background-secondary: var(--component-card-secondary);
+  --card-box-primary: 0 3px 3px rgb(0 0 0 / 15%), 0 -1px 1px rgb(0 0 0 / 1%);
+  --card-box-secondary: 0 -1px 1px rgb(0 0 0 / 5%), 0 3px 3px rgb(0 0 0 / 10%);
+  --card-box: var(--card-box-primary); //box shadow
+  --card-box-hover: 0 -1px 1px rgb(0 0 0 / 5%), 0 3px 3px rgb(0 0 0 / 20%);
+  --card-box-pressed: 0 -1px 1px rgb(0 0 0 / 5%), 0 1px 3px rgb(0 0 0 / 20%);
+  --card-headline: var(--color-foreground-text-strong);
+  --card-sub-headline: var(--color-foreground-text-subtle);
+  --card-body-color: var(--color-foreground-text-defined);
+  --card-divider: var(--color-foreground-border-discrete);
+  --card-icon-color: var(--color-foreground-icon-strong);
+}
 
-  .tds-mode-variant-primary {
-    --tds-card-box: var(--tds-card-box-primary);
+tds-card,
+.scania {
```

```suggestion
tds-card,
.scania tds-card {
```
Just to check if this is what we wanted?

---

Yes it is, must've missed it. 
It's fixed now 😄